database/things/abc.pldb

```diff
@@ -1,7 +1,12 @@
 title ABC
 appeared 1980
 type pl
-originCommunity	Centrum Wiskunde & Informatica 
+<<<<<<< HEAD
+originCommunity Centrum Wiskunde & Informatica 
+=======
```

Comment:
what's this?

Reply:
https://www.youtube.com/watch?v=dygYh2qsx64

Reply:
I am really sorry, i was concerned about test failure but that is never an excuse.

Reply:
Don't think. Just do.